src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -41,6 +49,17 @@ public FindCommand parse(String args) throws ParseException {
 
 
 
+    }
+
+    private String[] parseNameClassId(String args) throws ParseException {
+        String[] nameClassId = args.split("n/", 2)[1].split("c/", 2);
+
+        if (nameClassId.length < 2 || nameClassId[1].trim().isEmpty()) {
+            throw new ParseException(
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
+        }
+
+        return nameClassId[0].trim().split("\\s+");
     }
 
     private String[] parseName(String args) throws ParseException {
```

Comment:
Shouldn't parseName and parseClassId be abstracted out to ParserUtil? Referring to AddCommandParser

Reply:
Fixed